### ISSUE TITLE
past data not showing

### ISSUE THREAD

**Issue by rashearth** [nagix/chartjs-plugin-streaming#71]
Hi, thank you for your great support one again.
now, i have a question for inserting past time information on this streaming plugin.

This plugin could insert the future information, but not for the past?

Whenever i insert any past time data in the dataset, the array data disappears, and wont work out properly.
Is it impossible to insert past time data?
I want to make some realtime stock chart, and including past data is necessary.
Please kindly confirm.

**Comment by nagix** [comment]
The `ttl` option can be used to specify how long the data is retained. By setting it to a large number, you can prevent the past time data from being deleted. Then, you can show them by modifying the `duration` and/or `delay` options.

**Comment by rashearth** [comment]
Thanks for the prompt reply.
Actualy its not about how long it would retain.
I wanted to confirm if i could plot past time data.
But when i try it, i confirmed thst i could only plot future time, not for the past time.

Is there any way icould plot past data?

**Comment by nagix** [comment]
If you don't set `ttl`, the data points off the chart will be deleted immediately. So, `ttl` must be set if you want to keep the past time data.

**Comment by rashearth** [comment]
hi, thanks for your advise.
I have set the ttl to various numbers (including 10000000)
but the array all gets deleted when the plotting starts, and no plotting will happen for the past data.
the array will be looked like this( i think its deleted).
I have plotted down 30 past datas by using push function. ( i have done this by uing chart.update, or onrefresh function)

<img width="615" alt="スクリーンショット 2019-03-31 18 02 01" src="https://user-images.githubusercontent.com/26535295/55287066-65aa9000-53df-11e9-9a60-b2a19073c256.png">
